libs/pmp-web/repository/repositories-statistics/feature/src/lib/containers/repositories-statistics/repositories-statistics.component.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  OnDestroy,
+  OnInit
+} from '@angular/core';
+import { RepositoryStatisticsFacade } from '@pimp-my-pr/pmp-web/repository/data-access-repository-statistics';
+import { RepositoryStatistics } from '@pimp-my-pr/shared/domain';
+import { TableConfig } from '@pimp-my-pr/pmp-web/shared/domain';
+import { untilDestroyed } from 'ngx-take-until-destroy';
+import { RepositoriesStatisticsViewService } from './repositories-statistics-view.service';
+
+@Component({
+  selector: 'pimp-my-pr-repositories-statistics',
+  templateUrl: './repositories-statistics.component.html',
+  styleUrls: ['./repositories-statistics.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [RepositoriesStatisticsViewService]
+})
+export class RepositoriesStatisticsComponent implements OnInit, OnDestroy {
+  tableConfig: TableConfig<RepositoryStatistics[]>;
+
+  constructor(
+    private cdr: ChangeDetectorRef,
+    private repositoryStatisticsFacade: RepositoryStatisticsFacade,
+    private repositoriesStatisticsViewService: RepositoriesStatisticsViewService
```

Comment:
According to that post https://indepth.dev/lean-angular-components/ I think it can be good to name such a service as Presenter.

Reply:
As mentioned, this name would be better